canonical-web-and-design/ubuntu.com
# Update copy, add google cloud logo and link

## Done

- Updated page as per [copy doc](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/fips
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and make sure all requested changes have been made
- Check google logo link and make sure it routes to correct page

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5013
